### PR TITLE
Week1_Models_exercise_completed

### PR DESCRIPTION
**Pull request by mrani2020**
## Jumpstart Modules
Which week are you submitting completed practice exercises for? 

- [ X ] Week 1 (Models)
- [ ] Week 2 (Tests, Docs, Sources)
- [ ] Week 3 (Jinja, Macros, Packages)
- [ ] Week 4 (Materializations, Analyses, Seeds)

### CODE REVIEW

File: models/marts/core/dim_customers.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    group by 1
+),
+final as (
+    select
+        customers.customer_id,
+        customers.first_name,
+        customers.last_name,
+        customer_orders.first_order_date,
+        customer_orders.most_recent_order_date,
+        coalesce(customer_orders.number_of_orders, 0) as number_of_orders
```
Comment (ghost) [inline models/marts/core/dim_customers.sql]: doesn't have payment total yet

File: models/marts/core/dim_customers.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+with customers as (
+    select * from {{ ref('stg_customers')}}
+),
+orders as (
+    select * from {{ ref('stg_orders')}}
```
Comment (ghost) [inline models/marts/core/dim_customers.sql]: with a little refactoring of `fct_orders` and then ref'ing that here, you can get all the data points you need (including customer total payment amount)

File: models/marts/core/dim_customers.sql (hunk 1 above)
Comment (mrani2020) [reply models/marts/core/dim_customers.sql]: Added payment total.

File: models/marts/core/dim_customers.sql (hunk 2 above)
Comment (mrani2020) [reply models/marts/core/dim_customers.sql]: Updated to refactor fct_orders and then ref'ing that in dim_customers

**Pull request merged**